subcommand/server-acl-init/command.go

```diff
@@ -732,9 +741,9 @@ func (c *Command) withPrefix(resource string) string {
 	return fmt.Sprintf("%s-%s", c.flagResourcePrefix, resource)
 }
 
-// consulDatacenter returns the current datacenter name using the
+// consulDatacenter returns the current datacenter name and the primary datacenter using the
 // /agent/self API endpoint.
-func (c *Command) consulDatacenter(client *api.Client) (string, error) {
+func (c *Command) consulDatacenter(client *api.Client) (string, string, error) {
```

Comment:
Can we rename this to be a little more detailed about the datacenter being returned? eg `consulDatacenterList()` potentially?